WFD/swb_import.py

```diff
@@ -213,7 +216,9 @@ def make_significant_impact_type(self, data):
                * In both cases the reference can still be added.
 
         :param data: dict of data for a single swb
+        :return: [Statement]
         """
+        # @todo: Support for UNKN, OTHE (somevalue) + NOTA, NOSI (novalue)
```

Comment:
`novalue` would be used when it has been _decided_ that there are none.

The difference (also for `somevalue`) is between:
* No entry in Wikidata: Cannot say if it has been missed, it is none or it is unknown.
* `novalue` in Wikidata: The official data says none (for this year)
* `somevalue` in Wikidata: The official data says we are not sure (for this year)